logging/cluster-logging-release-notes.adoc

```diff
@@ -8,34 +8,65 @@ toc::[]
-* link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 - Bug Fix Advisory. Errata Advisory for Openshift Logging 5.0.0]
 
 [id="openshift-logging-supported-versions"]
 == Supported versions
 
-* OpenShift Logging version 5.0 runs on {product-title} versions 4.7 and 4.8.
-* OpenShift Logging version 5.1 runs on {product-title} versions 4.7 and 4.8.
+* OpenShift Logging versions 5.1 and 5.2 run on {product-title} versions 4.7 and 4.8.
 
 [id="openshift-logging-inclusive-language"]
 == Making open source more inclusive
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].
 
 // Release Notes by version
+
+[id="cluster-logging-release-notes-5-2-0"]
+== OpenShift Logging 5.2.0
+
+This release includes link:https://access.redhat.com/errata/RHSA-TBD[RHSA-TBD OpenShift Logging Bug Fix Release 5.2.0].
+
+[id="openshift-logging-5-2-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* TBD-description
+(link:https://issues.redhat.com/browse/LOG-TBD[LOG-TBD])
+
+* With this update, if you use the Fluentd forward protocol to forward log data over a TLS-encrypted connection, you can now use a password-encrypted private key file and specify the passphrase in the Cluster Log Forwarder configuration. For more information, see xref:../logging/cluster-logging-external.html#cluster-logging-collector-log-forward-fluentd_cluster-logging-external[Forwarding logs using the Fluentd forward protocol]
+(link:https://issues.redhat.com/browse/LOG-1525[LOG-1525])
+
+* With this update, you can use a username and password to authenticate a log forwarding connection to an external Elasticsearch instance. For example, if you cannot use mutual TLS (mTLS) because a third-party operates the Elasticsearch instance, you can use HTTP or HTTPS and set a secret that contains the username and password. For more information see xref:../logging/cluster-logging-external.adoc#cluster-logging-collector-log-forward-es_cluster-logging-external[Forwarding logs to an external Elasticsearch instance].
+(link:https://issues.redhat.com/browse/LOG-974[LOG-1022])
+
+
+
+[id="openshift-logging-5-2-0-deprecated-removed-features"]
+=== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
```

Comment:
There is a deprecated features module I think named `deprecated-feature.adoc` that you might want to use here instead?

Reply:
Thanks. These release notes are based on the release_notes/ocp-4-8-release-notes.adoc assembly, which doesn't include the modules/deprecated-feature.adoc module. It seems like we include this module to in assemblies for features that we are deprecating/removing. It is likely that I will probably remove this section from these release notes.